zap-known-issues.xml

```diff
@@ -82,53 +154,25 @@
   <sourceid>3</sourceid>
 </alertitem>
 <alertitem>
-  <pluginid>10023</pluginid>
-  <alert>Information Disclosure - Debug Error Messages</alert>
-  <name>Information Disclosure - Debug Error Messages</name>
-  <riskcode>1</riskcode>
-  <confidence>2</confidence>
-  <riskdesc>Low (Medium)</riskdesc>
-  <desc>&lt;p&gt;The response appeared to contain common error messages returned by platforms such as ASP.NET, and Web-servers such as IIS and Apache. You can configure the list of common debug messages.&lt;/p&gt;</desc>
-  <instances>
-  <instance>
-  <uri>https://ia-case-api-aat.service.core-compute-aat.internal/v2/api-docs</uri>
-  <method>GET</method>
-  <evidence>Internal Server Error</evidence>
-  </instance>
-  </instances>
-  <count>1</count>
-  <solution>&lt;p&gt;Disable debugging messages before pushing to production.&lt;/p&gt;</solution>
-  <reference>&lt;p&gt;&lt;/p&gt;</reference>
-  <cweid>200</cweid>
-  <wascid>13</wascid>
-  <sourceid>3</sourceid>
-</alertitem>
-<alertitem>
-  <pluginid>10015</pluginid>
-  <alert>Incomplete or No Cache-control and Pragma HTTP Header Set</alert>
-  <name>Incomplete or No Cache-control and Pragma HTTP Header Set</name>
-  <riskcode>1</riskcode>
-  <confidence>2</confidence>
-  <riskdesc>Low (Medium)</riskdesc>
-  <desc>&lt;p&gt;The cache-control and pragma HTTP header have not been set properly or are missing allowing the browser and proxies to cache content.&lt;/p&gt;</desc>
-  <instances>
-  <instance>
-  <uri>https://ia-case-api-aat.service.core-compute-aat.internal/v2/api-docs</uri>
-  <method>GET</method>
-  <param>Cache-Control</param>
-  </instance>
-  <instance>
-  <uri>https://ia-case-api-aat.service.core-compute-aat.internal/</uri>
-  <method>GET</method>
-  <param>Cache-Control</param>
-  <evidence>no-cache</evidence>
-  </instance>
-  </instances>
-  <count>2</count>
-  <solution>&lt;p&gt;Whenever possible ensure the cache-control HTTP header is set with no-cache, no-store, must-revalidate; and that the pragma HTTP header is set with no-cache.&lt;/p&gt;</solution>
-  <reference>&lt;p&gt;https://www.owasp.org/index.php/Session_Management_Cheat_Sheet#Web_Content_Caching&lt;/p&gt;</reference>
-  <cweid>525</cweid>
-  <wascid>13</wascid>
-  <sourceid>3</sourceid>
-</alertitem>
```

Comment:
I think we need to just give brief explanation or write comment why we are suppressing.

Reply:
ok, done